### PR TITLE
[Ready after merge of pull 98] Graduate reactions graphQL API to public

### PR DESCRIPTION
**Pull request by pjoviak**
Issue: [https://github.ibm.com/toscana/planning/issues/36710](https://github.ibm.com/toscana/planning/issues/36710)
Graduate reactions graphQL API to public.

### CODE REVIEW

**Comment by pjoviak** [comment]
@miguelestrada @jarusso @roy713 Please review.  Thank you.

File: Coming_Next.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-copyright: 'Copyright IBM Corp. 2017'
+copyright: 'Copyright IBM Corp. 2017,2018'
```
Comment (roy713) [inline Coming_Next.md]: Very minor but should we add a space?

File: Whats_New.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ Keep up to date on what's new in IBM Watson Work Services. We'll list the latest
 
 | Date          | What's new       |
 | ------------- |:-------------|
+| 11.Jul.2018   | Published [message reactions](./guides/V1_wwsg_Reactions.md). |
+| 11.Jul.2018   | Published [message deletion](./guides/V1_delete_message.md). |
+| 06.Jul.2018   | Added sections for [message editing](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_wwsg_EditingMessages.md) as `experimental`. |
```
Comment (roy713) [inline Whats_New.md]: Should we have experimental all caps?

File: guides/V1_message_main.md [hunk 3]
```diff
@@ -18,8 +18,9 @@ Watson Work Services represents a Message with this object.
...
-| reactions | [Reaction] | A set of optional objects that are added to a message, called reactions. Each reaction represents how users reacted to the message. A Reaction is represented by the reaction (String, usually an emoji but not required), the count (Int, number of users that reacted with the same reaction), and viewerHasReacted (Boolean, true, the calling user reacted with given reaction, or false, the calling user did not). This requires the `EXPERIMENTAL` value in `x-graphql-view header`. _Since this is an_ `EXPERIMENTAL` _capability, complete information can be found in our github repo, [see Coming Next section](../get-started/coming-next) for more info_. |
+| reactions | [Reaction] | A set of optional objects that are added to a message, called reactions. Each reaction represents how users reacted to the message. A Reaction is represented by the reaction (String, usually an emoji but not required), the count (Int, number of users that reacted with the same reaction), and viewerHasReacted (Boolean, true, the calling user reacted with given reaction, or false, the calling user did not). |
```
Comment (roy713) [inline guides/V1_message_main.md]: Should we decorate reaction, count, viewerHasReacted? I don't know what the standard in the docs is so...

**Review by roy713 — APPROVED** [review]

**Comment by pjoviak** [comment]
@miguelestrada @jarusso Could you please guardian review, when you get a chance ?
Thank you.

File: Coming_Next.md [hunk 4]
```diff
@@ -11,6 +11,6 @@ Here’s the stuff we’re working on next for you and IBM Watson Work Services.
...
+| **Editing a Message.** It's `EXPERIMENTAL` but you can check it out here, [Editing Messages](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_wwsg_EditingMessages.md). We have [Update Message](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_UpdateMessage.md) and changes to [Messages](./guides/V1_message_main.md) adding **contentUpdated** field and [Listening for Events](./guides/V1_wwsg_Webhooks.md) adding **message-edited**.              |
```
Comment (miguelestrada) [inline Coming_Next.md]: I think this should be separate PR (edit message API)
Actually even the graduation of Reaction and Delete message to public should be 2 separate ones, but will continue this review regardless.

File: NavigationMenu.md [hunk 5]
```diff
@@ -27,6 +27,18 @@
 * ## [Message](./guides/V1_message_main.md)
   * [Send a message into a conversation](./guides/V1_wwsg_Spaces.md)
   * [View a message from a conversation](./guides/V1_get_message.md)
+  * [Delete a message](./guides/V1_delete_message.md)
+  * [Events for message deletion (message-deleted)](./guides/V1_wwsg_Webhooks.md)
+  * [Optional inclusion of soft deleted flag with message retrieval](./guides/V1_message_main.md)
```
Comment (miguelestrada) [inline NavigationMenu.md]: Too long of a navigation title? @jarusso

File: NavigationMenu.md [hunk 6]
```diff
@@ -27,6 +27,18 @@
 * ## [Message](./guides/V1_message_main.md)
   * [Send a message into a conversation](./guides/V1_wwsg_Spaces.md)
   * [View a message from a conversation](./guides/V1_get_message.md)
+  * [Delete a message](./guides/V1_delete_message.md)
+  * [Events for message deletion (message-deleted)](./guides/V1_wwsg_Webhooks.md)
```
Comment (miguelestrada) [inline NavigationMenu.md]: Are we going to have a section for events related to each object or do we keep all events in one place @jarusso

File: guides/V1_message_main.md [hunk 7]
```diff
@@ -18,8 +18,9 @@ Watson Work Services represents a Message with this object.
 | createdBy   | Person    | Person who created this message |
 | updated     | Date    | Date this message  was updated |
 | updatedBy   | Person | Person who updated this message  |
-| softDeleted | Boolean | Indicates whether this message is soft-deleted. Soft-deleted messages will return null content and empty annotations. This requires the `EXPERIMENTAL` value in `x-graphql-view header`. |
+| softDeleted | Boolean | Indicates whether this message is soft-deleted. Soft-deleted messages will return null content and empty annotations. |
 | content   | String | The body of the message |
 | contentType | String | Mime type of the message content  |
+| contentUpdated | Date | Date this message content was edited or null. Available when you add `EXPERIMENTAL` to the `x-graphql-view` header |
```
Comment (miguelestrada) [inline guides/V1_message_main.md]: Related to update message work

**Review by miguelestrada — CHANGES_REQUESTED** [review]
@jarusso see my comments here.

**Comment by pjoviak** [comment]
@jarusso Comments?  @miguelestrada  Do you need this split this into multiple PRs?
Thank you.

File: NavigationMenu.md [hunk 8]
```diff
@@ -27,6 +27,18 @@
...
+  * [Optional inclusion of soft deleted flag with message retrieval](./guides/V1_message_main.md)
+  
+
+* ## [Reactions](./guides/V1_wwsg_Reactions.md)
```
Comment (miguelestrada) [inline NavigationMenu.md]: @pjoviak @jarusso so this reactions stuff is all related to messages and wondering if it should be inside the message section as well?

**Comment by miguelestrada** [comment]
@pjoviak  I think breaking it up should help speed up the delivery.  The 3 PRs would be:
1. Reactions
2. Delete Message
3. Update Message

@jarusso FYI

**Review by miguelestrada — APPROVED** [review]

**Pull request merged**